<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplaceWidget</class>
 <widget class="QWidget" name="FindReplaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>47</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>47</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>47</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>2</number>
   </property>
   <property name="verticalSpacing">
    <number>1</number>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Find:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>findLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QLineEdit" name="findLineEdit">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QToolButton" name="findPrevToolButton">
     <property name="toolTip">
      <string>Find Previous (Shift+F3)</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="QWorkBase.qrc">
       <normaloff>:/images/fr_find_prev.png</normaloff>:/images/fr_find_prev.png</iconset>
     </property>
     <property name="shortcut">
      <string>Shift+F3</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="2">
    <widget class="QToolButton" name="findNextToolButton">
     <property name="toolTip">
      <string>Find Next (F3)</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="QWorkBase.qrc">
       <normaloff>:/images/fr_find_next.png</normaloff>:/images/fr_find_next.png</iconset>
     </property>
     <property name="shortcut">
      <string>F3</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="2">
    <widget class="QCheckBox" name="caseSensetiveCheckBox">
     <property name="text">
      <string>&amp;Case Sensetive</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QCheckBox" name="useRegularExpressionsCheckBox">
     <property name="text">
      <string>Use Regular E&amp;xpressions</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5" rowspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QToolButton" name="replaceAllPushButton">
       <property name="text">
        <string>Replace All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="findInPushButton">
       <property name="text">
        <string>Find In ...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Replace:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>replaceLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="replaceLineEdit">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="replacePrevToolButton">
     <property name="toolTip">
      <string>Replace, Find Previous (Shift+F7)</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="QWorkBase.qrc">
       <normaloff>:/images/fr_repl_prev.png</normaloff>:/images/fr_repl_prev.png</iconset>
     </property>
     <property name="shortcut">
      <string>Shift+F7</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="replaceNextToolButton">
     <property name="toolTip">
      <string>Replace, Find Next (F7)</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="QWorkBase.qrc">
       <normaloff>:/images/fr_repl_next.png</normaloff>:/images/fr_repl_next.png</iconset>
     </property>
     <property name="shortcut">
      <string>F7</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QCheckBox" name="wholeWordsOnlyCheckBox">
     <property name="text">
      <string>&amp;Whole Words Only</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findLineEdit</tabstop>
  <tabstop>replaceLineEdit</tabstop>
  <tabstop>findPrevToolButton</tabstop>
  <tabstop>findNextToolButton</tabstop>
  <tabstop>replacePrevToolButton</tabstop>
  <tabstop>replaceNextToolButton</tabstop>
  <tabstop>caseSensetiveCheckBox</tabstop>
  <tabstop>useRegularExpressionsCheckBox</tabstop>
  <tabstop>wholeWordsOnlyCheckBox</tabstop>
 </tabstops>
 <resources>
  <include location="QWorkBase.qrc"/>
 </resources>
 <connections/>
</ui>
